packages/cli/src/commands/governance/approve-l2.test.ts

```diff
@@ -595,12 +602,15 @@ testWithAnvil('governance:approve cmd', (web3: Web3) => {
           })
         ).waitReceipt()
 
-        // setSecurityCouncil to 0x1B46C60e8B3B427d91d35df22FCfc3FF36407f35 (multisig address)
+        // setSecurityCouncil to multisig address
         await (
           await kit.sendTransaction({
             to: governance.address,
             from: DEFAULT_OWNER_ADDRESS,
-            data: '0x1c1083e20000000000000000000000001b46c60e8b3b427d91d35df22fcfc3ff36407f35',
+            // cast calldata "setSecurityCouncil(address)" <multisig-address>
+            data: `0x1c1083e2000000000000000000000000${multisig.address
+              .replace('0x', '')
+              .toLowerCase()}`,
```

Comment:
seems like the kind of thing that might be repeated alot eventually

Reply:
actually this one I don't really think so - it's very hotfix specific, but setting owner of the multisig might be something we just do as a global setup, I'm still considering it (among getting rid of the mnemonic alltogether), but wouldn't bother yet